src/constants.ts

```diff
@@ -7,9 +7,9 @@ export const PROGRAM_VERSION = packageDotJson.version;
 export const USER_AGENT = `CrossSeed/${PROGRAM_VERSION}`;
 
 export const EP_REGEX =
-	/^(?<title>.+?)[\s._](?<season>S\d+)?[_.\s]?(?<episode>E\d+(?:[-\s]?E?\d+)?)/i;
+	/^(?<title>.+?)[\s._](?=(?<season>S\d+)?[_.\s]?(?<episode>E\d+(?:[-\s]?E?\d+)?)|(?<date>\d{4}[\s._-]\d{2}[\s._-]\d{2}))/i;
 export const SEASON_REGEX =
-	/^(?<title>.+?)[_.\s](?<season>S\d+)(?:[.\-\s]*?(?<seasonmax>S?\d+))?(?=[_.\s](?!E\d+))/i;
+	/^(?<title>.+?)[_.\s](?=(?<season>S\d+)(?:[.\-_\s]*?(?<seasonmax>S?\d+))?(?=[_.\s](?!E\d+)))/i;
```

Comment:
Here would be left alone with the fix in #524, since the daily/dated shows do not change season pack identifications.

```suggestion
	/^(?<title>.+?)[_.\s-]+(?<season>S\d+)(?:[.\-\s_]*?(?<seasonmax>S?\d+))?(?=[_.\s](?!E\d+))/i;
```
**SEASON_REGEX** (https://regex101.com/r/fuE0bM/1)